### PR TITLE
Provide an address manually

### PR DESCRIPTION
**Pull request by irisfaraway**
https://eaflood.atlassian.net/browse/WC-101

While we prefer users to add addresses using the OS Places API, some users will not be able to use it, for example if they are overseas, if their address is not listed or if the service is down. These users should be able to fill in a separate form with details of their address.

### CODE REVIEW

**Comment by irisfaraway** [comment]
@Cruikshanks It's been a journey... happy to talk this through (more) in person. 😅

File: app/models/concerns/can_have_registration_attributes.rb [hunk 1]
```diff
@@ -37,5 +37,15 @@ module CanHaveRegistrationAttributes
     field :declaration,                                     type: Integer # Unsure of type
     field :regIdentifier, as: :reg_identifier,              type: String
     field :expires_on,                                      type: DateTime
+
+    def contact_address
```
Comment (Cruikshanks) [inline app/models/concerns/can_have_registration_attributes.rb]: Nice helpers! 👍

File: app/services/address_finder_service.rb [hunk 2]
```diff
@@ -20,7 +20,11 @@ def search_by_postcode
       :not_found
     rescue RestClient::ExceptionWithResponse => e
       Airbrake.notify(e)
-      Rails.logger.error "Os Places error: " + e.to_s
+      Rails.logger.error "OS Places error: " + e.to_s
```
Comment (Cruikshanks) [inline app/services/address_finder_service.rb]: Do you think there is any value tweaking the message for these errors to differentiate them in the log e.g. `OS Places response error` and `OS Places socket error`?

File: config/locales/forms/company_address_manual_forms/en.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+en:
+  company_address_manual_forms:
+    new:
+      heading_localAuthority: What's the address of the local authority or public body?
+      heading_limitedCompany: What's the address of the company?
+      heading_limitedLiabilityPartnership: What's the address of the limited liability partnership?
+      heading_partnership: What's the address of the partnership?
+      heading_soleTrader: What's the address of the individual?
```
Comment (Cruikshanks) [inline config/locales/forms/company_address_manual_forms/en.yml]: Needs to be *What's the address of the business?*

File: app/services/address_finder_service.rb (hunk 2 above)
Comment (irisfaraway) [reply app/services/address_finder_service.rb]: Sounds like a good idea to me. I'll tweak it now.

Comment (irisfaraway) [reply app/services/address_finder_service.rb]: @Cruikshanks Fixed!

**Pull request merged**